RecoTracker/TkHitPairs/src/CosmicHitPairGeneratorFromLayerPair.cc

```diff
@@ -107,7 +102,32 @@ void CosmicHitPairGeneratorFromLayerPair::hitPairs(const TrackingRegion& region,
     }
   }
 
-  //   stable_sort(allthepairs.begin(),allthepairs.end(),CompareHitPairsY(iSetup));
+  /*
+  class CompareHitPairsY {
+  public:
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant
... here it looks like a comment like `// uncomment if the sort operation commented out below needs to be called`

Reply:
I'm extending the multiline comment to include the sort part so it is obvious they belong together.